### ISSUE TITLE
flameshot - left-clicking on tray icon is ignored

### ISSUE THREAD

**Issue by jose1711** [netblue30/firejail#4924]
### Description

Clicking on the tray icon of flameshot should bring up a new dialog. This is however not happening if run via `firejail`.

### Steps to Reproduce

1. Run `flameshot`
2. Icon will appear in system tray
3. Click the icon
4. Another icon appears in system tray
5. If you click any of the icons (new/old) - nothing happens

### Expected behavior

Flameshot dialog should appear.

### Actual behavior

New icon/no further action.

### Behavior without a profile

`LC_ALL=C firejail --noprofile /usr/bin/flameshot` shows the same problem.
`/usr/bin/flameshot` - works

### Additional context

This problem definitely is not a deal-breaker but rather a minor annoyance as the icon can still be right-clicked which brings a context menu of the app.

### Environment

- Arch Linux, x86_64
- Firejail from git (03395e10b)

### Checklist

- [x] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [x] I can reproduce the issue without custom modifications (e.g. globals.local).
- [x The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [x] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [x] I have performed a short search for similar issues (to avoid opening a duplicate).
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
- [ ] I used `--profile=PROFILENAME` to set the right profile. (Only relevant for AppImages)

**Comment by rusty-snake** [comment]
FTR this works fine if you use StatusNotifierItem rather than the legacy XEmbed tray.

**Comment by jose1711** [comment]
> FTR this works fine if you use (K)StatusNotifierItem tray

That is correct, however if I use that then it works slightly differently. Instead of the dialog appearing instantly it opens a context menu
